prediction_market_agent/agents/logprobs_agent/deploy.py

```
@@ -0,0 +1,288 @@
+        return prediction_result
+
+    def _do_perplexity_search(self, market_question: str) -> PerplexityResponse:
+        return perplexity_search(
+            query=PERPLEXITY_QUERY.format(market_question=market_question),
+            api_keys=self.api_key,
+        )
+
+    def _process_links_parallel(
+        self, market_question: str, links: list[str], max_workers: int = 10
+    ) -> list[str]:
+        summaries: list[str] = []
```

Ruff (0.11.9)

Local variable `summaries` is assigned to but never used

Remove assignment to unused variable `summaries`

(F841)